### PR TITLE
AI now uses an updated lawset.

### PR DESCRIPTION
**Pull request by ScarKy0**
## About the PR

~~Updated the AI laws to use the lawset created in https://github.com/space-wizards/docs/pull/160~~
Updated the AI laws to use NT Default, media not relevant anymore.

## Why / Balance

While the Crewsimov lawset works well with borgs, I don't believe it works well with AI.

## Technical details

~~Created a new lawset called sAIDefault and added the corresponding circuit board, then assigned them to the Station AI by default.~~
Changed the default AI lawset to use NTDefault.

## Media

image

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

None.

**Changelog**

:cl: ScarKy0
- tweak: Station AI now uses a new lawset.

### CODE REVIEW

**Comment by ScarKy0** [comment]
Reposted because i messed something up im so sorry

**Comment by yellownes** [comment]
Terrible laws and a terrible order.
The AI should serve crew before it's own survival if needed, otherwise the AI could just bolt every single door to keep itself and the station in once piece.

**Comment by Gtheglorious** [comment]
XKCD_1613_the_three_laws_of_robotics

**Comment by ScarKy0** [comment]
Was told I can just make it NT laws.

**Comment by yellownes** [comment]
> Was told I can just make it NT laws.

More proof that maintainers neither play the game or have any actual sci-fi concepts

File: Resources/Prototypes/silicon-laws.yml [hunk 1]
```diff
@@ -161,6 +161,37 @@
...
+
+- type: siliconLawset
+  id: sAIDefault
+  laws:
+  - sAI1
+  - sAI2
+  - sAI3
+  - sAI4
+  obeysTo: laws-owner-crew
+
```
Comment (slarticodefast) [inline Resources/Prototypes/silicon-laws.yml:194]: I think this part isn't used anymore now that you replaced it with NTDefault

Comment (ScarKy0) [reply Resources/Prototypes/silicon-laws.yml:194]: Oops didnt see it, will fix it when i get the chance

**Comment by slarticodefast** [comment]
How did you even manage to get a merge conflict on a single line PR, lol
I think it might be that random whitespace that was added

**Comment by ScarKy0** [comment]
> How did you even manage to get a merge conflict on a single line PR, lol
I think it might be that random whitespace that was added

Its because my other PR that added laws got merged so the file changed, oops ill fix it soon

**Comment by slarticodefast** [comment]
Hey, I brought this up for dicussion with the maintainers and admins and due to a huge influx of similar PRs we decided to freeze AI law changes until some basic requirements for AI are implemented. See #31924 for more details. Once these are addressed the mentioned PRs will be re-evalutated. Thank you for your contribution.

**Pull request closed without merging**